cross/davfs2/Makefile

```diff
@@ -0,0 +1,32 @@
+PKG_NAME = davfs2
+PKG_VERS = 1.5.5
+PKG_EXT = tar.gz
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = http://download.savannah.gnu.org/releases/$(PKG_NAME)
+
+DEPENDS = cross/neon cross/libtool
+DESCRIPTION = Mount a WebDAV resource as a regular file system
+
+HOMEPAGE = http://savannah.nongnu.org/projects/davfs2
+LICENSE  = GPLv3
+
+export ssbindir=$(STAGING_INSTALL_PREFIX)/sbin/
```

Comment:
I wonder if it makes sense for a DSM package to distinct "bin" and "sbin" binaries... I would simplify and only deliver binaries in "bin"

Reply:
sbin folders are for commands that need superuser account to run - and mount with a type needs superuser.
But a short look into my DS-218+ shows that it is not unified here
```
/usr/bin/mount.cifs
/usr/sbin/mount.encryptfs
/usr/sbin/mount.nfs
/usr/sbin/mount.ntfs
```